main_2024.py

```diff
@@ -374,7 +412,16 @@ def main() -> int:
                 main_logger.debug(
                     "geolocation confidence: " + str(detection_world.confidence), True
                 )
```

Comment:
You can get rid of the geolocation logging to main now. Geolocation already has its own logging.